### PR TITLE
Update python.md

### PR DESCRIPTION
**Pull request by mssindhurid**
Initial release of azure-security-attestation Beta version for Python

### CODE REVIEW

**Comment by mssindhurid** [comment]
#sign-off

**Comment by mssindhurid** [comment]
@czubair  - Can you please review the changes for including Azure Attestation Python SDK in Jan release

**Comment by mssindhurid** [comment]
#sign-off

**Comment by alzimmermsft** [comment]
@xiangyan99 could you take a look

**Review by LarryOsterman — APPROVED** [review]

File: releases/2021-01/python.md [hunk 1]
```diff
@@ -35,6 +36,7 @@ pip install azure-eventhub-checkpointstoreblob-aio
 pip install azure-servicebus
 pip install azure-ai-textanalytics --pre
 pip install azure-ai-formrecognizer --pre
+pip install azure-security-attestation
```
Comment (xiangyan99) [inline releases/2021-01/python.md:39]: Do you need --pre?

Comment (mssindhurid) [reply releases/2021-01/python.md]: @xiangyan99  - added --pre as this is a Beta release

**Review by xiangyan99 — APPROVED** [review]

File: releases/2021-01/python.md [hunk 2]
```diff
@@ -66,6 +68,10 @@ If you have a bug or feature request for one of the libraries, please post an is
   - Fixed bug where connection establishment on macOS with Clang 12 triggering unrecognized selector exception (#15567).
   - Fixed bug in accessing message properties triggering segmentation fault when the underlying C bytes are NULL (#15568).
 
+### Microsoft Azure Attestation [Changelog](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/attestation/azure-security-attestation/CHANGELOG.md#100b1-2020-12-09)
+
+  - Initial release of azure-security-attestation Beta version to support data-plane operations of Microsoft Azure Attestation.
```
Comment (czubair) [inline releases/2021-01/python.md:73]: You can condense to: Initial release of Microsoft Azure Attestation

**Review by czubair — APPROVED** [review]

File: releases/2021-01/python.md (hunk 2 above)
Comment (mssindhurid) [reply releases/2021-01/python.md]: @czubair  - updated to "Initial release of Microsoft Azure Attestation"

**Pull request merged**